index.html

```diff
@@ -546,6 +546,12 @@ <h2 class="page__title">Tooling</h2>
 
       <li class="tooling__item"><div class="tooling__item--wrap">
         <span class="tooling__ico"><span class="tooling__ico__logo tooling__ico--construction"></span></span><!--
+     --><h3 class="tooling__title"><a href="https://www.npmjs.com/package/grunt-blueprint-validator" class="block larger"><span class="block">Blueprint API Validator</span><span class="tooling__link"></span></a> <span class="block smaller">Grunt Plugin</span></h3><!--
```

Comment:
Can you add this at the last please? We list them in first-come-first-serve basis.